mix.exs

```diff
@@ -22,7 +22,7 @@ defmodule VerkWeb.Mixfile do
 
   def application do
     [mod: {VerkWeb, []},
-     env: [{VerkWeb.Endpoint, [http: [port: 4000], cache_static_manifest: "priv/static/manifest.json", server: true]}],
+     env: [{VerkWeb.Endpoint, [http: [port: 4000], server: false]}],
```

Comment:
Ok let's keep it without the static_manifest. I will keep providing it inside the hex package so the user can enable it if wanted. :+1: